active_directory_ldap/komand_active_directory_ldap/connection/schema.py

```diff
@@ -16,6 +17,13 @@ class ConnectionSchema(komand.Input):
   "type": "object",
   "title": "Variables",
   "properties": {
+    "auto_referrals": {
+      "type": "boolean",
+      "title": "Auto Referrals",
+      "description": "Allows the plugin to follow referrals from the specified Active Directory server to other Active Directory servers",
```

Comment:
maybe change to "chase referrals"?  Just a thought because that's the language Microsoft seems to use

Reply:
We should match the same language then. @r7-mgorny